core/templates/social-metadata.tid

```diff
@@ -27,12 +27,12 @@ tags: $:/tags/RawMarkupWikified
 <<meta-plain "og:type" "website" "property">>
 <<meta-wikified "og:title" "$:/SiteTitle" "property">>
 <<meta-wikified "og:description" "$:/SiteSubtitle" "property">>
-<<meta-plain "og:image" "$:/SitePreviewImageUrl" "property">>
+<<meta-plain "og:image" "$:/SitePreviewUrl" "property">>
 
 <!-- Twitter Meta Tags -->
 <<meta-plain "twitter:card" "summary_large_image">>
 <<meta-plain "twitter:domain" "$:/SiteDomain" "property">>
 <<meta-plain "twitter:url" "$:/SiteUrl" "property">>
 <<meta-wikified "twitter:title" "$:/SiteTitle">>
 <<meta-wikified "twitter:description" "$:/SiteSubtitle">>
-<<meta-plain "twitter:image" "$:/SitePreviewImageUrl">>
+<<meta-plain "twitter:image" "$:/SitePreviewUrl">>
```

Comment:
Compare with:
* https://github.com/TiddlyWiki/TiddlyWiki5/blob/master/core/ui/ControlPanel/SocialCard.tid#L20 and 
* https://github.com/TiddlyWiki/TiddlyWiki5/blob/master/core/ui/ControlPanel/SocialCard.tid#L13